packages/eslint-plugin/test/fixtures/types/expect-diagnostics-versioned/.eslintrc.js

```diff
@@ -0,0 +1,10 @@
+module.exports = {
+  overrides: [
+    {
+      files: ["*.ts", "*.cts", "*.mts", "*.tsx"],
+      rules: {
+        "@definitelytyped/expect": ["error", { versionsToTest: [{ versionName: "x.y", path: "typescript" }] }],
```

Comment:
how does this test work? Does it treat "x.y" as a real Typescript version somehow?

Reply:
This is just a pretty name, the ones we generate when creating this list in `dtslint`. It's used instead of the version provided by `typescript`. Here I'm just giving it a random name since I don't want to hardcode/reuse the version from the current repo as it changes (and would break baselines).